src/test/java/AccountMapperTest.java

```diff
@@ -14,22 +13,14 @@
 import static org.mockito.Mockito.lenient;
 
 @ExtendWith(MockitoExtension.class)
-public class AccountMapperTest {
+public class AccountMapperTest extends ShareableDataForTests {
```

Comment:
This is wrong use of inheritance. Child class should be a subclass of the parent, not just use the parameters.